src/kbmod/__init__.py

```diff
@@ -1,15 +1,18 @@
 import warnings
 
 try:
-    from ._version import version as __version__
+    from ._version import version as __version__  # noqa: F401
 except ImportError:
     warnings.warn("Unable to determine the package version. " "This is likely a broken installation.")
 
-# This is needed for compatibility with some older compilers: erfinv needs to be
-# imported before other packages though I'm not sure why.
-from scipy.special import erfinv
```

Comment:
When this line is removed, the tests no longer run on Baldur for me. I get the error:

```
  File "/astro/users/jkubica/kbmod_logs/kbmod/src/kbmod/__init__.py", line 14, in <module>
    from kbmod.search import Logging
ImportError: /astro/users/jkubica/kbmod_logs/kbmod/src/kbmod/search.cpython-312-x86_64-linux-gnu.so: undefined symbol: _ZNSt15__exception_ptr13exception_ptr10_M_releaseEv
```
But as soon as I put it back in all the tests pass fine.  I have no idea why or what is happening . But the PR works fine with the include.

Reply:
I followed the chain of dependencies back and it turns out there is an easy fix. If we include `import warnings` the error also goes away. It looks like this was being imported through one of the `__init__.py` files in scipy.

Can you add `import warnings` to the PR in place of this scipy import?

Reply:
Now that the PR is merged, I'm seeing the same error on baldur. Though need for a rollback since

`from scipy.special import erfinv` in `__init__.py` does seem to fix it for me as well. 

Wondering however if a better solution for the environment issue was found?